.vscode/launch.json

```diff
@@ -4,6 +4,21 @@
     // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
     "version": "0.2.0",
     "configurations": [
+        {
+            "name": "Flutter: Attach to Device",
+            "type": "dart",
+            "request": "attach"
+        },
+        {
+            "name": "Run Mobile Dev",
+            "program": "lib/main.dart",
+            "request": "launch",
+            "flutterMode": "debug",
+            "type": "dart",
+            "args": [
+                "--dart-define=LENRA_SERVER_URL=https://api.staging.lenra.io"
+            ]
+        },
         {
             "name": "Run Web Dev",
             "program": "lib/main.dart",
```

Comment:
This one is then not needed because when you're in VSCode with the `Run Mobile Dev` selected, you have a button to select your target (Android, iOS, Web, Chrome, etc...). It can be used to run on chrome like the `Run Web dev` you have.